crates/cli/Cargo.toml

```diff
@@ -30,7 +30,9 @@ name = "benchmark"
 harness = false
 
 [features]
+default = ["qjs-rt"]
```

Comment:
Should we enable it by default?

Reply:
1MB is actually a lot but that's not my point.

Considering this is still experimental in a way, we shouldn't enable it by default (except in our release binaries).
After polishing it throughout 0.26, we can enable it in 0.27 and maybe make it the default in 0.28 (or 1.0?).

Reply:
Just for the sake of clarity: setting this flag has no effect on the `ts g src/grammar.json` codepath, right?